src/language-html/embed.js

```diff
@@ -135,6 +136,17 @@ function embed(path, options) {
 
     case "front-matter":
       return (textToDoc) => printFrontMatter(node, textToDoc);
+
+    // TODO: This should work on `blockParameter`, but currently we don't have a parser for it.
+    case "block":
```

Comment:
We should either wrap parameters in a node or find out how to parse them separately. We'll see how to fix it later.

Reply:
@DingWeizhe Can you fix this?

**Prettier pr-15606**
[Playground link](https://deploy-preview-15606--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEABOBbADjAnsAXwB0oT0AbAZzgAIBLAMxsJABoQIc7pLlQBDAE6CIAdwAKQhLxT9yo-rl7sARoP5gA1nBgBlfhjgAZOlDjIGc6qvVadurBtMBzZDEEBXOO2oY6bz28QShdyOABFDwh4CysggCtKAA9dUIiomKRLKiCARwy4cREsGRB+SgBaMzgAE1q2EHd+OnIXAGEIDAx+ZDLycgaQqGcwgEEYdzoVD3hxOEETM1ic9gALGAxyAHVVunhKRzA4XWk9ugA3PdxesEplEHOvAEkoOthdMEE6HFHX3TwwstrCAsCJqFt1FheqC4NRBOdzOxTHCYEV+M5ukCgo5BHDevxhh5yEIGqDTDAtnQajBVsgABwABnYgjg+ToLLRGJ6WTi7Bg-BUlOptKQACZ2B5qAAVAUybLAzAqWp1GpGAnODzouAAMQggm6Exc+JmEBABAIQA)
<!-- prettier-ignore -->
```sh
--parser angular
```

**Input:**
<!-- prettier-ignore -->
```html
@empty{}

@else if {}
```

**Output:**
<!-- prettier-ignore -->
```html
@empty {}

} @else if {}

```